src/js/common/components/Modal.jsx

```diff
@@ -78,7 +78,7 @@ class Modal extends React.Component {
     }
 
     return (
-      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={`${id}-title`}>
+      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={id && title ? `${id}-title` : undefined}>
```

Comment:
OK, but how can we make sure the `aria-labelledby` attribute has a valid value if neither `id` nor `title` is defined? 

Also, did you want the value to be `${id}-${title}` or `${id}-title`?